src/main/fc/cli.c

```diff
@@ -2862,15 +2862,15 @@ static void cliResource(char *cmdline)
             const char* owner;
             owner = ownerNames[ioRecs[i].owner];
 
-            cliPrintf("%c%02d: %s ", IO_GPIOPortIdx(ioRecs + i) + 'A', IO_GPIOPinIdx(ioRecs + i), owner);
+            cliPrintf("%c%02d: %s", IO_GPIOPortIdx(ioRecs + i) + 'A', IO_GPIOPinIdx(ioRecs + i), owner);
```

Comment:
@hydra Sorry, I believe this is how it should be. Attempt to share the following `"%d"` by adding a space here will make an extra space at the end of the line, which is semantically incorrect.